data/com.canonical.Unity8.gschema.xml

```diff
@@ -72,4 +72,12 @@
       <description>If too many unsuccessful login attempts have been made in a row, the greeter locks you out for a while.  This unix timestamp indicates when you were locked out, so unity8 knows when to let you back in.</description>
     </key>
   </schema>
+
```

Comment:
I suggest when adding new gsettings use the com.lomiri userspace to ease with transition later on.

Reply:
Is that possible when we're in the com.canonical.Unity.gschema.xml file?

Reply:
Alright, now there's a new com.lomiri.gschema.xml file. @wdehoog, is the change in this file needed any more?